app/views/places/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1 align="center"> List of places where you training </h1>
+<% @places.each do |places| %>
+  <div class="row col-xs-8 col-xs-offset-2">
+    <div class="well well-lg">
+      <div class="places-name">
+       
```

Comment:
empty div with no content should be an one-liner

Reply:
OK